web-ui/src/components/reports-section/CheckinReport.css

```diff
@@ -1,42 +1,83 @@
-#accordion-checkin-date {
+.checkin-report-card {
+  display: flex;
   flex-direction: column;
-}
+  width: 100%;
 
-.checkin-report-link {
-  display: flex;
-  align-items: center;
-  margin-bottom: 0.5rem;
-}
+  .checkin-report-card-title-container {
+    display: flex;
+    flex-direction: column;
+  }
 
-.checkin-report-link .MuiChip-colorPrimary {
-  height: 1.5rem;
-  margin-left: 0.5rem;
-}
+  .checkin-report-card-name {
+    overflow: hidden;
+    white-space: nowrap;
+    text-overflow: ellipsis;
+    max-width: 20rem;
+  }
 
-.checkin-report-link .MuiChip-colorSecondary {
-  height: 1.5rem;
-  margin-left: 0.5rem;
-}
+  .checkin-report-card-actions {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    gap: 1rem;
+  }
 
-.MuiAccordionSummary-content {
-  align-items: center;
-}
+  .member-sub-card {
+    background-color: #fafafa;
+    margin-top: 1rem;
+    padding: 0.5rem;
 
-.MuiAccordionSummary-content > p {
-  margin-left: 1rem;
-}
+    .member-sub-card-accordion-details {
+      display: flex;
+      flex-direction: column;
+      margin: 0.5rem 0;
+    }
 
-#member-sub-card {
-  background-color: var(--checkins-palette-background-default);
-  margin-top: 1rem;
-}
+    .member-sub-card-summmary-content {
+      display: flex;
+      flex: 1;
+      align-items: center;
+      margin-right: 1rem;
+
+      &:first-child {
+        margin-top: 0;
+      }
+
+      p {
+        flex: 1;
+        margin: 0.5rem;
+      }
+
+      .last-connected {
+        color: #a0a0a0;
+        /* background-color: #f44336; */
```

Comment:
Finished with the commented lines?

Reply:
This review started before I was ready. This comment is outdated.